pcf/particle/aws/dynamodb/dynamodb_table.py

```diff
@@ -98,11 +100,22 @@ def get_status(self):
 
     def _start(self):
         """
+        Converts the Tag attribute syntax if the key exists
         Starts the dynamodb particle that matches the desired state function
 
         Returns:
             reponse of boto3 create_table
         """
+        # convert the definition Tag to the appropriate Tags=["Key": "string", "Value": "string"] syntax
+        if self.desired_state_definition.get("Tags"):
```

Comment:
not sure if we need this. we can make the user already have the tags in this format since it would come from the boto docs. for our generic tagging particle we can start playimg around with how we want to simplify it.

Reply:
We have a couple of particles which follow the pattern of having users define Tags with this format: `"Tags":{"key": "value"}` and then convert the syntax to the appropriate boto3 form, i.e. `S3Particle`. For the sake of keeping consistency I opted to continue that pattern. We should pick one way and update any other particles to maintain this consistency. Would you like me to create another issue and work on that? 

Side question: Do you have any idea why boto3 has all these different syntaxes of tagging resources? It is so bizarre.

Reply:
yea its annoying that that it's different for all the particles. But I agree we should keep it consistent with boto docs. We should prob change for s3 too if its not consistent with boto3.